engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/generic/sources.scala

```diff
@@ -25,8 +27,15 @@ object sources {
 
   import collection.JavaConverters._
 
-  class GenericJsonSourceFactory(processObjectDependencies: ProcessObjectDependencies) extends KafkaSourceFactory[java.util.Map[_, _]](
-    JsonMapDeserialization, None, JsonRecordFormatter, processObjectDependencies)
+  class EspValueDeserializaitionSchemaFactory[T](schema: DeserializationSchema[T]) extends ConsumerRecordDeserializationSchemaFactory[String, T]{
```

Comment:
Can you name it more descriptive like `FixedValueDeserializaitionSchemaFactory` and move it to the same package as `ConsumerRecordDeserializationSchemaFactory`? Also can you rename `T` to `V` and `schema` to `valueSchema`?